livre/eq/render/shaders/fragRayCast.glsl

```diff
@@ -161,10 +161,6 @@ void main( void )
 
             if( localResult.a > EARLY_EXIT )
                 break;
-
-            if( i > nSamplesPerRay )
```

Comment:
I also think that this check can be removed; However that makes i unused, it should be removed from the loop too.